docs/develop/08_opendid/03_opendid_service.md

```diff
@@ -0,0 +1,115 @@
+---
+id: opendid_service
+title: Run OpenDID Service
+---
+
+<!-- TODO: Overview and steps -->
+
+## Configuration
+
+Running the OpenDID service requires some configuration and a DID.
+The DID establishes a secure session with an identity wallet using a key agreement key of type `X25519KeyAgreementKey2019` included in the DID Document generated by the setup container.
+
+OpenDID serves a [well-known DID configuration](https://identity.foundation/.well-known/resources/did-configuration/), which the identity wallet uses to ensure that the domain is linked to the specified DID.
+
+### Run setup container
+
+Before running the `opendid-setup` container, set two environment variables:
+
+1. `SEED` to provide an account with funds (minimum of 3 KILT) for the DID generation.
+
+    ```bash
+    export SEED="dont try this seed its completely made up for this nice example"
+    ```
+
+2. `ENDPOINT`
+
+    Set to "spiritnet" if the account is on the spiritnet production network.
+
+    ```bash
+    export ENDPOINT="spiritnet"
+    ```
+
+    Set to "peregrine" if the account is on the peregrine test network.
+
+    ```bash
+    export ENDPOINT="peregrine"
+    ```
+
+    Then run the setup with the following command:
+
+    ```bash
+    docker run --rm -it -e "ENDPOINT=${ENDPOINT}" -v $(pwd):/data docker.io/kiltprotocol/opendid-setup:latest "${SEED}"
+    ```
+
+The command generates a set of new mnemonics and then derives a DID from them and generates multiple files into the current directory:
+
+1. `config.yaml` The configuration file used by the OpenDID service.
+2. `did-secrets.json` This file contains the public and secret keys in the DID Document.
+
+    :::warning
+    Keep a secure backup of this file as it contains all the secret keys.
+    :::
+
+3. `did-document.json` contains the DID Document generated by this setup.
+
+    :::danger
+    You only need the `config.yaml` to run the OpenDID service.
+    This file includes the generated mnemonic and secret keys and you should protect it from unauthorized access.
```

Comment:
The DID Document? Doesn't it contain only public info?

Reply:
No this is the config file… @abdulmth ?

Reply:
Correct, the sentence here is talking about the `config.yaml` which contains the seed.

Reply:
I think I was tricked by the indentation. If the danger applies to the list as a whole, then it should live in the top level. In this case it seems to apply to the point about point 3 `did-document.json`.